pkg/storaged/dialog.jsx

```diff
@@ -684,6 +687,25 @@ export const SelectSpace = (tag, title, options) => {
     };
 };
 
+const CheckBoxComponent = ({ tag, val, title, options, update_function }) => {
+    return (
+        <div key={tag} className="checkbox">
+            <label key={tag}>
+                <input type="checkbox" data-field={tag} data-field-type="checkbox"
+                       checked={val}
+                       onChange={event => update_function(event.target.checked)} />
+                {title}
+            </label>
+            { options.tooltip && <OverlayTrigger overlay={ <Tooltip id="tip-service">{options.tooltip}</Tooltip> } placement="right" >
+                <a className="dialog-item-tooltip">
+                    <span className="fa fa-lg fa-info-circle" />
+                </a>
+            </OverlayTrigger>
+            }
+        </div>
+    );
+};
+
 export const CheckBox = (tag, title, options) => {
```

Comment:
Let's remove `CheckBox` altogether.  The remaining uses are "Store passphrase" in format-dialog.jsx, and the VDO options.  The latter need to be a field set anyway.